### DISCUSSION TITLE
Coverage Metric threshold can affect a Space Health status

### DISCUSSION THREAD

**Discussion by munderseth** [testspace-com/support#12] [General]
A Space publishing [Code Coverage](https://help.testspace.com/docs/dashboard/space-metrics#code-coverage) has a default `Threshold` setting of **50%**.  If the coverage is below this threshold the Results will be marked as `Unhealthy`. 

The `Threshold` can be updated using the [hamburger menu](https://help.testspace.com/docs/dashboard/space-metrics#code-coverage).  

All new branches created will inherit the Codec Coverage Metric `Threshold` from the base branch.  

@gersmann I thought you might be interested in this discussion.

**Comment by aminbs** [comment]
Hello @munderseth, thank you for letting us know! we did find the menu, but is there a way to change the value for all spaces ?

**Reply by munderseth** [reply]
Hi @aminbs there is no way to manually change all of the spaces at one time. Sorry about that.  

If a branch is created from another branch (base) that already contains the Metrics in Testspace it will inherit it automatically. 

A couple of notes:
- You can *remove* the threshold altogether. Thus the badge will **not** affect the health (commit status) - no green or red colors. 
- If a space is set up for the code coverage metric, and does **not** receive the file, the Results will be marked as unhealthy

**Reply by aminbs** [reply]
Excellent that solves things for us, thank you @munderseth !
